src/main/kotlin/ai/digital/integration/server/deploy/tasks/kube/scanning/KubeAwsScannerTask.kt

```diff
@@ -0,0 +1,61 @@
+package ai.digital.integration.server.deploy.tasks.kube.scanning
+
+import ai.digital.integration.server.common.constant.PluginConstant
+import ai.digital.integration.server.common.util.*
+import ai.digital.integration.server.common.util.KubeScanningUtil
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+import java.io.File
+
+open class KubeAwsScannerTask : DefaultTask() {
+
+    companion object {
+        const val NAME = "kubeAwsScanner"
+        const val reportFile = "aws-eks"
+    }
+
+    init {
+        group = PluginConstant.PLUGIN_GROUP
+        this.dependsOn(CheckingOutKubeBenchTask.NAME)
+        this.finalizedBy(KubeAwsScannerFinalizerTask.NAME)
+
+    }
+
+    private fun kubeBenchPushAndApply() {
+        project.exec {
+            executable = "cd"
+            args = listOf(KubeScanningUtil.getKubeBenchDir(project))
+        }
+        ProcessUtil.execute(project, "aws", listOf("ecr", "create-repository", "--repository-name", "k8s/kube-bench", "--image-tag-mutability", "MUTABLE"), false)
+
+        ProcessUtil.executeCommand(
+                "aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin ${KubeScanningUtil.getAWSAccountId(project)}")
+        ProcessUtil.execute(project, "docker", listOf("build", "-t", "k8s/kube-bench", KubeScanningUtil.getKubeBenchDir(project)), false)
+        ProcessUtil.execute(project, "docker", listOf("tag", "k8s/kube-bench:latest", "${KubeScanningUtil.getAWSAccountId(project)}/k8s/kube-bench:latest"), false)
+        ProcessUtil.execute(project, "docker", listOf("push", "${KubeScanningUtil.getAWSAccountId(project)}/k8s/kube-bench:latest"), false)
+
+        updateKubeBenchImage()
+
+        KubeCtlUtil.apply(File("${KubeScanningUtil.getKubeBenchDir(project)}/job-eks.yaml"))
+
+
+    }
+
+    private fun updateKubeBenchImage() {
```

Comment:
Please format code. There is a hot key for that in Intellij IDEA
![Screenshot 2021-12-06 at 10 50 56](https://user-images.githubusercontent.com/2265094/144824974-398bb49c-196f-4a0e-ae73-ed142e5819ea.png)